specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2019-06-15/bms.json

```diff
@@ -1321,13 +1321,6 @@
             "modelAsString": true
           }
         },
-        "healthDetails": {
```

Comment:
removing property is a breaking change

Reply:
Hi @changlong-liu  and @akning-ms ,

The properties which are being removed had never been enabled for the customer. We had done the Swagger and SDK changes for adding these properties and enums as part of a recent PR (https://github.com/Azure/azure-rest-api-specs/pull/9711) and the service code to use these fields is not enabled. So no customer can possibly be using these properties. Based on internal feedback we have decided to follow a different approach so we wont be needing these properties anymore. So i would like to remove them. Even the enum "HealthStatus" needs to be removed/updated. If you can agree upon this then i can remove the enum also as part of this PR instead of a separate PR.

Thanks,
Labonya

Reply:
As discussed with @akning-ms I have removed the enums.
Will update the SDK for this change along with the changes for a few more Swagger PRs which are currently in our pipeline.